package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Conditional expresstion util for server/client",
   "main": "./dist/index.js",
   "scripts": {
-    "test": "gulp &&./node_modules/.bin/mocha --reporter spec",
-    "prepublish": "gulp"
+    "test": "mocha --compilers js:babel-core/register --reporter spec",
```

Comment:
Also, no need to make dists before running tests. Tests are run against the source, and run through babel transpiler on test instead.